src/Tribe/Events/Views/V2/Hooks.php

```diff
@@ -89,7 +122,8 @@ protected function add_actions() {
 	 * @since 4.10.9
 	 */
 	protected function add_filters() {
-		add_filter( 'tribe_template_path_list', [ $this, 'filter_template_path_list' ] );
+		add_filter( 'tribe_template_path_list', [ $this, 'filter_template_path_list' ], 15, 2 );
+		add_filter( 'tribe_template_origin_namespace_map', [ $this, 'filter_add_template_origin_namespace' ], 15, 3 );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: aff5357e6ce610d654fe1e9ac02d27ca</sub>